requirements.txt

```
@@ -1,4 +1,5 @@
-pulpcore>=3.4
+pulpcore>=3.6
```

```suggestion
pulpcore>=3.6.0.dev0
```
I needed this to pip install it with current pulpcore master.